chase/frontend/components/dashboard/timer.tsx

```diff
@@ -0,0 +1,120 @@
+import React, { useEffect, useRef } from "react";
+import WidgetTemplate from "./widgettemplate";
+import { Toast } from "primereact/toast";
+
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faGavel } from "@fortawesome/free-solid-svg-icons";
+import { faComments } from "@fortawesome/free-solid-svg-icons";
+import { faCoffee } from "@fortawesome/free-solid-svg-icons";
+import { faCirclePause } from "@fortawesome/free-solid-svg-icons";
+
+export default function TimerWidget({
+  headline,
+  until,
+  category,
+}: {
+  headline: string;
+  until: Date | null;
+  category: "formal" | "informal" | "pause" | "suspension";
```

Comment:
We should use a typescript enum for this as soon as we have a state management solution